src/composables/useLeaflet.ts

```diff
@@ -92,7 +92,9 @@ export const useLeaflet = (
                 // map while the bounds were locked, now we track each layer that will open a
                 // tooltip and close them if they are not the most recent layer to make sure
                 // no old tooltips remain open
-                layerWithOpenTooltip.value?.closeTooltip();
+                if (layer !== layerWithOpenTooltip.value) {
+                    layerWithOpenTooltip.value?.closeTooltip();
+                }
```

Comment:
Presumably you could include line 98 in the if clause too - no point in setting layerWithOpenTooltip value to layer if you've already check that it is...